onnx/test/shape_inference_test.py

```diff
@@ -2993,6 +2993,22 @@ def test_negative_log_likehood_input_weight_shape_mismatch(self):  # type: () ->
             [])
         self.assertRaises(checker.ValidationError, self._inferred, graph)
 
+    def test_mean_square_none(self):  # type: () -> None
+        graph = self._make_graph(
+            [("x", TensorProto.FLOAT, (2, 3)),
+             ("y", TensorProto.FLOAT, (2, 3))],
+            [make_node('MeanSquaredDistance', ['x', 'y'], ['z'], reduction='none')],
+            [],)
+        self._assert_inferred(graph, [make_tensor_value_info('z', TensorProto.FLOAT, (2, 3))])  # type: ignore
+
+    def test_mean_square_mean(self):  # type: () -> None
+        graph = self._make_graph(
+            [("x", TensorProto.FLOAT, (2, 3)),
```

Comment:
Please also add 1-D and 4-D cases. Thank you.

Reply:
there is already 1-D, I will add 4-D